kong/tools/responses.lua

```diff
@@ -152,11 +152,16 @@ local function send_response(status_code)
                                   {message = content})
       if not encoded then
         ngx.log(ngx.ERR, "[admin] could not encode value: ", err)
+        ngx.header["Content-Length"] = 0
 
       else
         ngx.header["Content-Type"] = "application/json; charset=utf-8"
-        ngx.say(encoded)
+        ngx.header["Content-Length"] = #encoded
+        ngx.print(encoded)
```

Comment:
@bungle Could you elaborate on this change? It certainty seems concerning to me...